shell/renderer/api/electron_api_context_bridge.cc

```diff
@@ -348,6 +349,7 @@ v8::Local<v8::Value> ProxyFunctionWrapper(
     size_t func_id,
     bool support_dynamic_properties,
     gin_helper::Arguments* args) {
+  TRACE_EVENT_INSTANT0("electron", "ContextBridge::ProxyFunctionWrapper");
```

Comment:
Why is this an instant rather than measuring a duration?

Reply:
The duration isn't actually the cost of this method, the duration would include the duration of the V8 function actually being called.  I didn't think it was good to give the misleading impression that `ProxyFunctionWrapper` was slow when it was actually the underlying method.

Thoughts?

Reply:
That should also be visible in the trace via the separate v8 function call tracing. I think this should be a duration. Instants are for things which are instant, which this is not. It's generally the case that durations in tracing include other things, and the tracing UI is equipped to illustrate that.

Reply:
Fair enough, updated 👍